standalone/inc/pup/PUPTrigger.cpp

```diff
@@ -76,7 +76,8 @@ PUPTrigger::PUPTrigger(bool active, const string& szDescript, const string& szTr
 PUPTrigger* PUPTrigger::CreateFromCSV(PUPScreen* pScreen, string line)
 {
    vector<string> parts = parse_csv_line(line);
-   if (parts.size() != 14) {
+   if (parts.size() != 14)
```

Comment:
I think the brace formatting got changed on this file. { after if's etc.

Reply:
Is there some kind of command I can run to validate formatting or even better reformat?

Reply:
unfortunately no, it's more just stick with the rest of the code base for consistency. sorry